Add email variants offering mentoring to all teams (#78)
srobo/team-emails: SR2021/2021-02-14-league-2-followup-inactive.md

---
to: Teams who attended the leagues but aren't very active
subject: Would you like a mentor?
---

Hi,

It was great to see your robot at the second league session. We hope you found
it as exciting as we did and are looking forward to the new challenges which
module Ⅲ brings. The full rulebook and simulator updates are available on our
website, so do check them out if you haven't already!

We also wanted to check on how you're doing and whether there's any more help
you need?

In addition to Discord, we'd also like to offer you some more hands-on support
in the form of a regular mentor. This would be a volunteer who is able to be
around whenever you regularly meet and would get to know your team and your
robot. Whatever platform you use (whether it's Discord or something else),
they'd be able to join your meetings to help you out. (For safeguarding reasons

> we need your team leader to also be in any video calls; your team leader will
> have details about this).

Reviewer: ```suggestion
we need you to also be in any video calls).
```

Author: c6bca27 fixes this for all three emails.

---
to: Teams who attended the leagues but aren't very active
subject: Would you like a mentor?
---

Hi,

It was great to see your robot at the second league session. We hope you found
it as exciting as we did and are looking forward to the new challenges which

> module Ⅲ brings. The full rulebook and simulator updates are available on our

Reviewer 2: Is it worth adding links to the full rulebook and simulator page inline?

Author: d9242d7e36b95c8a715e80bf895478aa4909413d
